host_core/lib/host_core/web_assembly/imports.ex

```diff
@@ -2,6 +2,8 @@ defmodule HostCore.WebAssembly.Imports do
   require Logger
   alias HostCore.Actors.ActorModule.State
 
+  @wasmcloud_logging "wasmcloud:builtin:logging"
+
```

Comment:
we already mentioned it in slack but we just have to remember to change wash and tell others to re-sign